scene/main/scene_tree.h

```diff
@@ -90,8 +90,8 @@ class SceneTree : public MainLoop {
 	Window *root = nullptr;
 
 	uint64_t tree_version = 1;
-	double physics_process_time = 1.0;
-	double process_time = 1.0;
+	double physics_process_time = 0;
+	double process_time = 0;
```

Comment:
We try to write doubles with the decimal too to be explicit, so this should be `0.0`.